ansible/templates/minigraph_png.j2

```diff
@@ -11,6 +11,9 @@
         <EndPort>{{ vm_intfs[if_index] }}</EndPort>
         <StartDevice>{{ inventory_hostname }}</StartDevice>
         <StartPort>{{ port_alias[dut_intfs[if_index]] }}</StartPort>
+{% if device_conn[inventory_hostname][port_alias_map[port_alias[dut_intfs[if_index]]]] is defined %}
```

Comment:
Will this always have 'speed' ?
Can we use port_speed instead, which is used to populate asic front panel interfaces information?
{% if port_alias[loop.index - 1] in port_speed %}
        <Bandwidth>{{ port_speed[port_alias[loop.index - 1]] }}</Bandwidth>
{% else %}

Reply:
@SuvarnaMeenakshi port_speed is based of port_config.ini. We want to set port actual connection speed which is define in device connection.